### ISSUE TITLE
mirnylib versiion

### ISSUE THREAD

**Issue by qixin271** [mirnylab/mirnylib-legacy#2]
The result is 
 Name                    Version                   Build  Channel
mirnylib                  0.0.0                    pypi_0    pypi
What is the correct mirnylib version number?

**Comment by mimakaev** [comment]
The library is deprecated and has not been updated in 5 years.

**Comment by qixin271** [comment]
thanks！



---- Replied Message ----
| From | Max ***@***.***> |
| Date | 12/14/2024 09:59 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [mirnylab/mirnylib-legacy] mirnylib versiion (Issue #2) |

The library is deprecated and has not been updated in 5 years.

**Comment by qixin271** [comment]
Excuse me for disturbing you,may I ask if hiclib can still be used for data analysis?



---- Replied Message ----
| From | Max ***@***.***> |
| Date | 12/14/2024 09:59 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [mirnylab/mirnylib-legacy] mirnylib versiion (Issue #2) |

The library is deprecated and has not been updated in 5 years.

**Comment by mimakaev** [comment]
Nope, I won't recommend really. Checkout open2c - distiller and cooltools.

**Comment by qixin271** [comment]
Thanks a ton！



---- Replied Message ----
| From | Max ***@***.***> |
| Date | 12/14/2024 17:28 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [mirnylab/mirnylib-legacy] mirnylib versiion (Issue #2) |

Nope, I won't recommend really. Checkout open2c - distiller and cooltools.
